src/System.IO.FileSystem/src/System/IO/Enumeration/FileSystemEnumerable.cs

```diff
@@ -24,22 +24,27 @@ public class FileSystemEnumerable<TResult> : IEnumerable<TResult>
         private readonly string _directory;
 
         public FileSystemEnumerable(string directory, FindTransform transform, EnumerationOptions options = null)
+            : this(directory, transform, isNormalized: false, options)
+        {
+        }
+
+        internal FileSystemEnumerable(string directory, FindTransform transform, bool isNormalized, EnumerationOptions options = null)
```

Comment:
You should add the new parameter to the end and not set default values on the internal one.

Reply:
Ok thanks, will do.